docs/topics/settings.rst

```diff
@@ -384,7 +384,7 @@ Default: ``0``
 
 Scope: ``scrapy.spidermiddlewares.depth.DepthMiddleware``
 
-An integer that is used to adjust the :attr:`~scrapy.http.Request.priority` of
+An integer that is used to adjust the priority of
```

Comment:
What was the reason to change it?

Reply:
There is no such attribute in documentation https://github.com/scrapy/scrapy/blob/master/docs/topics/request-response.rst